BAT/BCAux.h

```diff
@@ -130,7 +130,11 @@ class BCTrash
 public:
     BCTrash() {}
     BCTrash(const BCTrash<T>&) {}
-    BCTrash<T>& operator=(const BCTrash<T>&) {}
+    BCTrash<T>& operator=(const BCTrash<T>&)
+    {
+        return *this;
+    }
+
```

Comment:
No it must copyable. This way I can avoid having to write a copy ctor for BCFitter et al. The point of this class is  to keep root objects alive so plotting works but nonetheless we want to remove these things from heap. I don't want to transfer the lifetime to a copy. You could argue that then the swap shouldn't do anything either but I marginally prefer that the pointer are copied in that case.